Rakefile

```diff
@@ -23,12 +24,15 @@ Thread.abort_on_exception = true
 
 begin
   require 'rspec/core/rake_task'
+  require 'rubocop/rake_task'
+
+  RuboCop::RakeTask.new
 
   desc "Run specs"
   RSpec::Core::RakeTask.new(:spec) do |t|
-    t.rspec_opts = %w(-fs --color)
-    #t.ruby_opts  = %w(-w)
+    t.rspec_opts = %w(--color)
   end
+  task :spec => :rubocop
   task :default => :spec
```

Comment:
> and have it execute via that after the rest of the tests have run

So, if I understand correctly, desire is to follow pretty common practice:

- `rake spec` runs `spec`
- `rake rubocop` runs `rubocop`
- `rake` runs `spec` and then in case of success `rubocop`

Your patch makes:

- `rake spec` runs `rubocop` and then in case of success `spec`
- `rake rubocop` runs `rubocop`
- `rake`  runs `rubocop` and then in case of success `spec`